src/fcontext/fcontext.c

```diff
@@ -4,6 +4,21 @@
 #include <stdlib.h>
 #include <assert.h>
 
+// Lifted from https://github.com/llvm/llvm-project/blob/release/16.x/clang/lib/Headers/cet.h
+# ifdef __LP64__
+#  if __CET__ & 0x1
+#    define _CET_ENDBR __asm("endbr64")
+#  else
+#    define _CET_ENDBR
```

Comment:
The empty stub definition should be `#define _CET_ENDBR ((void)0)`. This forces the compiler to require a semicolon after the macro and silences potential warnings about "extraneous" semicolons.

Reply:
Are you sure about this one? I don't get any complain with or without a definition after `_CET_ENDBR`.

Reply:
It depends on your compiler and warning flags. I find it to be good practice even if there is no warning, just to make the compiler consistently require the semicolon.